components/BookmarkIcon/BookmarkIcon.js

```diff
@@ -5,17 +5,25 @@ export default function BookmarkIcon({ active, handleClick }) {
   return (
     <>
       {active ? (
-        <BookmarkDash size={25} onClick={() => handleClick()} />
+        <BookmarkDash onClick={() => handleClick()} />
       ) : (
-        <BookmarkHeart size={25} onClick={() => handleClick()} />
+        <BookmarkHeart onClick={() => handleClick()} />
       )}
     </>
   );
 }
 
 const BookmarkDash = styled(BsBookmarkDash)`
   cursor: pointer;
+  font-size: 25px;
+  @media screen and (min-width: 700px) {
+    font-size: 33px;
+  }
 `;
 const BookmarkHeart = styled(BsBookmarkHeart)`
   cursor: pointer;
+  font-size: 25px;
+  @media screen and (min-width: 700px) {
+    font-size: 33px;
+  }
 `;
```

Comment:
`BookmarkHeart` and `BookmarkDash` seem identical to me, would it make more sense to have only one styled component instead of 2?